samples/dynamically_add_module_sample/README.md

```diff
@@ -2,8 +2,6 @@
 
 This document provides a detailed overview of the Dynamically Add Module sample [code](./src) which uses the fundamental components of the Azure IoT Gateway SDK architecture to create a gateway without modules, adding modules (and links) from a JSON file after the gateway is created, starting a gateway, removing a module and destroying a gateway.
 
-
-
 #Dev box setup
```

Comment:
You need a space after the `#` or else the markdown doesn't display correctly in GitHub. Same for the rest of this document.